src/main/java/com/yahoo/bullet/record/BulletRecord.java

```diff
@@ -259,141 +121,126 @@ public Object get(String field, int index) {
         return casted.get(index);
     }
 
-    /**
-     * Returns true iff the given top-level field exists in the record.
-     *
-     * @param field The field to check if it exists.
-     * @return A boolean denoting whether there was a mapping for the field.
-     */
-    public boolean hasField(String field) {
-        forceFailIfCannotRead();
-        return data.containsKey(field);
-    }
+    // ******************************************** SETTERS ********************************************
 
     /**
-     * Returns the number of fields in the record.
+     * Insert a boolean field.
      *
-     * @return An int representing the number of fields stored.
+     * @param field The non-null name of the field.
+     * @param object The value to insert.
+     * @return this object for chaining.
```

Comment:
Can we make consistent whether we capitalize or not the first sentence for a return?